lib/restaurant.rb

```diff
@@ -0,0 +1,40 @@
+require 'twilio-ruby'
+require_relative 'twilio_credentials'
+
+class Restaurant
+
+  WAIT_TIME = 3600 # in seconds
+
+  def initialize
+    @account_sid = TWILIO_ACCOUNT_SID
+    @auth_token = TWILIO_AUTH_TOKEN
+    @from = FROM
+  end
+
+  def place(order, number = nil)
+    @order = order
+    deadline = wait.strftime('%H:%M')
+    message = "Thank you! Your order was placed and will be delivered before #{deadline}"
+    return message if number == nil?
+    send_sms(message, number)
+  end
+
+  private
+
```

Comment:
You mentioned in your reflection that you could have broken some of your solution down more. I suspect you're talking about this bit, where an SMS is sent...?  That would make sense.  You could use a `Notification` class..?